tasking/manager.go

```diff
@@ -128,6 +127,7 @@ func (m *Manager) startReady() {
 				Log.Info("Task postponed.", "id", ready.ID)
 			} else {
 				err := task.Run()
+				Log.Trace(err)
```

Comment:
Just curious if the Log.trace should be called in all cases (regardless to the err), otherwise looks good to me!

Reply:
Yes, for simplicity.  The Log.Trace() will no-op when err == nil.